Makefile.am

```diff
@@ -0,0 +1,37 @@
+ACLOCAL_AMFLAGS = -I m4
+
+dist_sbin_SCRIPTS = ddclient
+
+EXTRA_DIST = \
+	COPYING \
+	COPYRIGHT \
+	ChangeLog.md \
+	README.cisco \
+	README.md \
+	README.ssl \
+	TODO \
+	UPGRADE \
+	autogen \
+	sample-ddclient-wrapper.sh \
+	sample-etc_cron.d_ddclient \
+	sample-etc_ddclient.conf \
+	sample-etc_dhclient-exit-hooks \
+	sample-etc_dhcpc_dhcpcd-eth0.exe \
+	sample-etc_ppp_ip-up.local \
+	sample-etc_rc.d_init.d_ddclient \
+	sample-etc_rc.d_init.d_ddclient.alpine \
+	sample-etc_rc.d_init.d_ddclient.lsb \
+	sample-etc_rc.d_init.d_ddclient.redhat \
+	sample-etc_rc.d_init.d_ddclient.ubuntu \
+	sample-etc_systemd.service \
+	sample-get-ip-from-fritzbox
+
+LOG_DRIVER = env AM_TAP_AWK='$(AWK)' $(SHELL) \
+	$(top_srcdir)/build-aux/tap-driver.sh
+TEST_EXTENSIONS = .pl
+PL_LOG_DRIVER = $(LOG_DRIVER)
+PL_LOG_COMPILER = $(PERL)
+AM_PL_LOG_FLAGS = -Mstrict -w -I'$(abs_top_srcdir)'
+TESTS = \
+	t/load.pl
```

Comment:
I think this should be replaced with something that just loads all files in ``t/``.

Reply:
That would be nice, but Automake intentionally does not support globs.